packages/template-retail-react-app/app/commerce-api/hooks/useBasket.js

```diff
@@ -56,7 +56,7 @@ export default function useBasket(opts = {}) {
              * to interact with a customer basket. All other calls are done through the
              * ShopperBaskets API, which in our case, uses OCAPI rather than commerce sdk.
```

Comment:
Please update the jsdoc here accordingly.

Reply:
Re-iterating the comment above:

I ended up implementing deferring baskets because the double auth init in commerce-sdk-react was breaking my basket initialization. I fixed that bug in commerce-sdk-react in https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2629 . I no longer need to defer baskets and I've reverted all changes in the useBasket hooks back to original. This also significantly cuts down the change surface making it easier for customers to adopt the upgrade.

Thanks a lot for catching this.